### DISCUSSION TITLE
Diversify rate

### DISCUSSION THREAD

**Discussion by alserembani94** [farhan-helmy/ratemysurau#6] [Ideas]
Kalau boleh, buat banyak rate system (atau categorise rate system) 

Contoh:

General - rate surau secara keseluruhan
Kebolehcapaian - untuk rate accessibility, how mesra is for the OKU
Kemudahan - untuk rate kemudahan, macam tempat wudhu, sejadah, ruang solat, parking
Imam - rate imam
Komuniti - rate how active komuniti surau tu, contoh aktif ceramah ke
Sekuriti - rate berapa selamat, contoh berpagar ke, ada CCTV ke, surau tu ada open close time ke, ada guard ke

**Comment by farhan-helmy** [comment]
Idea ni nice gilaaaa, tapi bab ui aku tak reti sangat, nak letak for each category ada row of star rating dia sendiri ke, kalau ada 5 dan keatas nanti ada 5 row

**Reply by alserembani94** [reply]
Good point 👍🏾

Mungkin way of kita nak improve is, provide ni sebagai optional, atau decide on data structure whether kita nak fix amount of rate category.

Boleh jugak nak improve dari segi UI, contohnya collapsible rating window, so takde la content tu congested sangat.
